__TESTS__/query-string.spec.tsx

```diff
@@ -0,0 +1,164 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+/* eslint-enable import/no-extraneous-dependencies */
+import React from 'react';
+
+import QueryString from '../pages/query-string';
+
+jest.mock('react-i18next', () => ({
```

Comment:
If we add translations for the input field, you will probably need to remove this mock and use `renderWithI18n` from `__TESTS__/helper` instead of `render` from `@testing-library/react`.

Working on a better way to handle this if you have any ideas (that would be a separate PR if you wanted to take it on)

Reply:
My solution was to use the translation field instead of the English text (i.e. `screen.getByRole('textbox', { name: 'queryString:parameter 1' })` instead of `screen.getByRole('textbox', { name: 'Parameter 1' })`)
Not sure if that's a better solution or not, but I think one advantage is that it works even if the translated text gets changed later? Let me know what you think, I can switch to using renderWithI18n instead if you'd like so that there's better consistency.

Reply:
Yeah, I thought about that, my only concern in doing it that way is if the translation slug is wrong the test will still pass while it is technically not working as expected. But, now that you mention it, TypeScript will catch the missing translation slug, so let's leave it as is, I'll update the other tests and move the mock to `jest.config.js`. Thanks!